CONTRIBUTING_GUIDELINES.md

```diff
@@ -0,0 +1,27 @@
+# Contributing Guidelines  
+  
+This documentation contains a set of guidelines to help you during the contribution process.
+ 
+We are happy to welcome all the contributions from anyone willing to improve this project. Thank you for helping out and remember, **no contribution is small.** 
+ 
+- Participants are expected to follow project guidelines and coding style.
+- Each participant/contributor will be assigned **2 issues (max)** at a time to work on.
+- Participants/contributors have to give an update about their work in every **3 days** while completing issues. If not then the issue will be assigned to other.
```

Comment:
Please change the time interval required as mentioned in the readme.

Reply:
![image](https://user-images.githubusercontent.com/76895975/122239617-b5050a00-cede-11eb-9955-4f2f0a40c0ae.png)

Reply:
yep